### ISSUE TITLE
Elaborate experimental FP16 mode of MPBidirectWinograd 

### ISSUE THREAD

**Issue by atamazov** [ROCm/MIOpen#521]
Each configuration should be controlled separately. Group convolution needs to be double checked. Then 16/16 F(2,3) and F(3,3) to be tested on a network level. I guess those could be enabled for normal use.

_Originally posted by @Kirpich30000 in https://github.com/ROCmSoftwarePlatform/MIOpen/pull/491#issuecomment-710587689_

**Comment by ppanchad-amd** [comment]
@atamazov Is this ticket still relevant?

**Comment by atamazov** [comment]
@ppanchad-amd Yes, the effect of enabing `MIOPEN_DEBUG_AMD_MP_BD_WINOGRAD_EXPEREMENTAL_FP16_TRANSFORM` envvar it not investigated yet, and this is quite amount of work to do.

**Comment by JonathanLichtnerAMD** [comment]
Closing since MIOpen is being migrated to [ROCm/rocm-libraries](https://github.com/ROCm/rocm-libraries).  Please re-open there if you believe this is still an issue.
